scripts/test/TOFTOF/TOFTOFGUITest.py

```diff
@@ -0,0 +1,133 @@
+from __future__ import (absolute_import, division, print_function)
+
+from reduction_gui.reduction.toftof.toftof_reduction import TOFTOFScriptElement, OptionalFloat
+from reduction_gui.widgets.toftof.toftof_setup import TOFTOFSetupWidget
+import mantid
```

Comment:
Please, run `flake8` on the test files: it says, for example, that `mantid` is an unused import.